config_parser.cc

```diff
@@ -210,7 +211,10 @@ bool NginxConfigParser::Parse(std::istream* config_file, NginxConfig* config) {
         // Error.
         break;
       }
-      return true;
+      if(config_stack.size() == 1)
```

Comment:
Additionally, you should follow the style guide (implied) for this repo, this line should look like
if (config_stack.size() == 1) {
  return true;
} else {
  break;
}